class.jetpack-connection-banner.php

```diff
@@ -217,16 +235,11 @@ class="jp-wpcom-connect__hide-phone-and-smaller"
 
                             <div class="jp-banner__button-container">
                                 <span class="jp-banner__tos-blurb"><?php jetpack_render_tos_blurb(); ?></span>
-								<?php if ( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME' ) ) : ?>
-                                    <div class="jp-banner__button-container">
-                                    </div>
-								<?php else: ?>
-                                    <a
-                                            href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>"
-                                            class="dops-button is-primary">
-										<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
-                                    </a>
-								<?php endif; ?>
+                                <a
+                                        href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>"
+                                        class="dops-button is-primary jp-connect-button">
+									<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
```

Comment:
Nit: I realize this was here before, but something appears to be going on with the indentation here.

Reply:
I checked on master and there's no misalignment there, I think is github fault here.